// @girs/gnome-shell/dist/ui/index.cjs
module.exports.accessDialog = require('./accessDialog.cjs');
module.exports.altTab = require('./altTab.cjs');
module.exports.animation = require('./animation.cjs');
module.exports.appDisplay = require('./appDisplay.cjs');
module.exports.appFavorites = require('./appFavorites.cjs');
module.exports.appMenu = require('./appMenu.cjs');
module.exports.audioDeviceSelection = require('./audioDeviceSelection.cjs');
module.exports.background = require('./background.cjs');
module.exports.backgroundMenu = require('./backgroundMenu.cjs');
module.exports.barLevel = require('./barLevel.cjs');
module.exports.boxpointer = require('./boxpointer.cjs');
module.exports.calendar = require('./calendar.cjs');
module.exports.checkBox = require('./checkBox.cjs');
module.exports.closeDialog = require('./closeDialog.cjs');
module.exports.components = require('./components.cjs');
module.exports.dateMenu = require('./dateMenu.cjs');
module.exports.dialog = require('./dialog.cjs');
module.exports.dnd = require('./dnd.cjs');
module.exports.extensionSystem = require('./extensionSystem.cjs');
module.exports.iconGrid = require('./iconGrid.cjs');
module.exports.layout = require('./layout.cjs');
module.exports.lightbox = require('./lightbox.cjs');
module.exports.lookingGlass = require('./lookingGlass.cjs');
module.exports.main = require('./main.cjs');
module.exports.messageList = require('./messageList.cjs');
module.exports.messageTray = require('./messageTray.cjs');
module.exports.modalDialog = require('./modalDialog.cjs');
module.exports.mpris = require('./mpris.cjs');
module.exports.notificationDaemon = require('./notificationDaemon.cjs');
module.exports.panel = require('./panel.cjs');
module.exports.panelMenu = require('./panelMenu.cjs');
module.exports.popupMenu = require('./popupMenu.cjs');
module.exports.quickSettings = require('./quickSettings.cjs');
module.exports.ripples = require('./ripples.cjs');
module.exports.search = require('./search.cjs');
module.exports.slider = require('./slider.cjs');
module.exports.switcherPopup = require('./switcherPopup.cjs');
module.exports.userWidget = require('./userWidget.cjs');
module.exports.windowManager = require('./windowManager.cjs');
module.exports.windowPreview = require('./windowPreview.cjs');
module.exports.workspace = require('./workspace.cjs');
module.exports.workspaceAnimation = require('./workspaceAnimation.cjs');
module.exports.workspacesView = require('./workspacesView.cjs');
module.exports.Components = require('./components/index.cjs');
module.exports.Status = require('./status/index.cjs');
