// @girs/gnome-shell/dist/ui/status/index.cjs
module.exports.accessibility = require('./accessibility.cjs');
module.exports.autoRotate = require('./autoRotate.cjs');
module.exports.backgroundApps = require('./backgroundApps.cjs');
module.exports.backlight = require('./backlight.cjs');
module.exports.bluetooth = require('./bluetooth.cjs');
module.exports.brightness = require('./brightness.cjs');
module.exports.camera = require('./camera.cjs');
module.exports.darkMode = require('./darkMode.cjs');
module.exports.dwellClick = require('./dwellClick.cjs');
module.exports.keyboard = require('./keyboard.cjs');
module.exports.location = require('./location.cjs');
module.exports.network = require('./network.cjs');
module.exports.nightLight = require('./nightLight.cjs');
module.exports.powerProfiles = require('./powerProfiles.cjs');
module.exports.remoteAccess = require('./remoteAccess.cjs');
module.exports.rfkill = require('./rfkill.cjs');
module.exports.system = require('./system.cjs');
module.exports.thunderbolt = require('./thunderbolt.cjs');
module.exports.volume = require('./volume.cjs');
